packages/web-components/src/utils/utils.ts

```diff
@@ -26,9 +27,9 @@ export function getSlottedNodes(
     ? root.shadowRoot.querySelector('slot').assignedNodes()
     : root.shadowRoot.childNodes;
 
-  return Array.from(children).filter(
+  return nodeName !== null ? Array.from(children).filter(
```

Comment:
I don't think it needs to be done in this PR since `getSlottedNodes` doesn't currently have any unit tests, but now that we're using it more and making changes to it, we probably want a ticket for adding test coverage.